api/lib/infrastructure/repositories/certification-challenge-repository.js

```diff
@@ -33,38 +22,35 @@ module.exports = {
       courseId: certificationCourse.id,
     });
 
-    return certificationChallenge.save()
-      .then((certificationChallenge) => {
-        return _toDomain(certificationChallenge);
-      });
+    const savedCertificationChallenge = await certificationChallenge.save();
+    return bookshelfToDomainConverter.buildDomainObject(CertificationChallengeBookshelf, savedCertificationChallenge);
   },
 
-  findByCertificationCourseId(certificationCourseId) {
-    return CertificationChallengeBookshelf
+  async findByCertificationCourseId(certificationCourseId) {
```

Comment:
J'ai du mal à voir dans ces cas là l'avantage à utiliser async/await alors que l'API de Bookshelf se prête bien à faire la conversion dans un `then` à la fin de la requête.

Reply:
L'homogénéité avec le reste du code en général, et en particulier avec la fonction suivante où il y a plus de traitements à appliquer après le retour de Bookshelf.

Reply:
:+1: